api/src/main/java/org/openmrs/module/providermanagement/ProviderPersonRelationship.java

```diff
@@ -0,0 +1,29 @@
+package org.openmrs.module.providermanagement;
+
+import org.openmrs.Person;
+import org.openmrs.Relationship;
+import org.openmrs.RelationshipType;
+
+public class ProviderPersonRelationship {
+
+    Person person = null;
+    String identifier = null;
+    Integer objectId = null;
+    Relationship relationship= null;
+    RelationshipType relationshipType = null;
+
+    public ProviderPersonRelationship() {}
+
+    public ProviderPersonRelationship(Person person,
```

Comment:
Thanks @mogoodrich . I have moved this class into a new org.openmrs.module.providermanagement.relationship package. Please let me know if we should call it something else.